src/api/composition/CompositionCollection.js

```diff
@@ -56,10 +56,10 @@
  */
 export default class CompositionCollection {
   domainObject;
-  #provider;
-  #publicAPI;
-  #listeners;
-  #mutables;
+  provider;
```

Comment:
Why do we need to make these public?

Reply:
To make a long story short, [Proxies and private members don't play well together](https://github.com/tc39/proposal-class-fields/issues/106).